docs/en/community/team.md

```diff
@@ -0,0 +1,39 @@
+# Apollo Team
+
+The Apollo team is comprised of Members and Contributors. Members have direct access to the source of Apollo project and actively evolve the code-base. Contributors improve the project through submission of patches and suggestions to the Members. The number of Contributors to the project is unbounded. All contributions to Apollo are greatly appreciated, whether for trivial cleanups, big new features or other material rewards.
+
+For more information about the community governance model, please refer [GOVERNANCE.md](https://github.com/ctripcorp/apollo/blob/master/GOVERNANCE.md).
+
+## Members
+
+Members include Project Management Committee members and committers. The list is in alphabet order.
+
+### Project Management Committee
+
+| Github ID  | Name       | Organization |
+| ---------- | ---------- | ------------ |
+| JaredTan95 | Jared Tan  | DaoCloud     |
+| kezhenxu94 | Zhenxu Ke  | Tetrate      |
+| nobodyiam  | Jason Song | Ant Group    |
+
+### Committer
+| Github ID   | Name          | Organization |
+| ----------- | ------------- | ------------ |
+| Anilpole    | Xiaoquan Wang |              |
```

Comment:
@Anilople @klboke @nisiyong @vdisk-group 

please help to update the corresponding content.